modules/exploits/linux/local/cve_2022_0995_watch_queue.rb

```diff
@@ -0,0 +1,162 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GreatRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::System
+  include Msf::Post::Linux::Compile
+  include Msf::Post::Linux::Kernel
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Watch Queue Out of Bounds Write',
+        'Description' => %q{
+          This module exploits a vulnerability in the Linux Kernel's watch_queue event notification system. It relies on a heap
+          out-of-bounds write in kernel memory. The exploit fails occasionally, but it does not crash the target and can be run
+          multiple times until privilege escalation is achieved.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'bonfee',     # PoC
+          'bwatters-r7' # Aka @tychos_moose, Metasploit Module
```

Comment:
```suggestion
          'Jann Horn', # discovery and poc
          'bonfee', # exploit
          'bwatters-r7' # Aka @tychos_moose, Metasploit Module
```

Reply:
Recommended change implemented in https://github.com/rapid7/metasploit-framework/pull/16422/commits/78d4ac85926baee5d9cd4e6cd67415a8d3d8858f